<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetEventSettings</class>
 <widget class="QWidget" name="WidgetEventSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  color: white;
}
QGroupBox {
  border-radius: 0px;
  border: 3px solid rgb(131, 147, 167);
  background-color: transparent; /*rgb(80,112,125);*/
}
QLineEdit {
  background-color: rgb(101,117, 137);  
}
QTextEdit {
  background-color: rgb(101,117, 137);  
}
QPlainTextEdit {
  background-color: rgb(101,117, 137);  
}
QComboBox {
  background-color: rgb(101,117, 137);  
}
QCheckBox::indicator {
  border: 2px solid rgb(80,112,125);
}
QCheckBox::indicator:checked {
  background-color: white;
}
QListView {
  background-color: rgb(101,117, 137);  
}
QTableWidget {
  background-color: rgb(101,117, 137);  
}
QDateTimeEdit {
  background-color: rgb(101,117, 137);  
}
QPushButton {
  background: rgb(101,117, 137);
}
QPushButton::checked {
  border: 2px solid white;
  font-weight: bold;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="bottomMargin">
               <number>6</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Event Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditName">
                  <property name="maxLength">
                   <number>32</number>
                  </property>
                  <property name="placeholderText">
                   <string>Event name</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxIsPublic">
                  <property name="toolTip">
                   <string>Public events are open to any user.</string>
                  </property>
                  <property name="text">
                   <string>Event is public.</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="labelPhoto">
                <property name="minimumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="pixmap">
                 <pixmap resource="../resources/application.qrc">:/icon-group.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignHCenter|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="textEditDescription">
              <property name="placeholderText">
               <string>A short event description</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Event Start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dateTimeEditStart">
              <property name="displayFormat">
               <string>dd.MM.yyyy - HH:mm</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Repeated Event</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Week Days</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QPushButton" name="pushButtonWDMon">
                     <property name="focusPolicy">
                      <enum>Qt::NoFocus</enum>
                     </property>
                     <property name="text">
                      <string>Mon</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButtonWDTue">
                     <property name="focusPolicy">
                      <enum>Qt::NoFocus</enum>
                     </property>
                     <property name="text">
                      <string>Tue</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButtonWDWed">
                     <property name="focusPolicy">
                      <enum>Qt::NoFocus</enum>
                     </property>
                     <property name="text">
                      <string>Wed</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButtonWDThu">
                     <property name="focusPolicy">
                      <enum>Qt::NoFocus</enum>
                     </property>
                     <property name="text">
                      <string>Thu</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButtonWDFri">
                     <property name="focusPolicy">
                      <enum>Qt::NoFocus</enum>
                     </property>
                     <property name="text">
                      <string>Fri</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButtonWDSat">
                     <property name="focusPolicy">
                      <enum>Qt::NoFocus</enum>
                     </property>
                     <property name="text">
                      <string>Sat</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButtonWDSun">
                     <property name="focusPolicy">
                      <enum>Qt::NoFocus</enum>
                     </property>
                     <property name="text">
                      <string>Sun</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Day Time</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="timeEditDayTime">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Members</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QTableWidget" name="tableWidgetMembers">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                 <property name="showGrid">
                  <bool>false</bool>
                 </property>
                 <attribute name="horizontalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderHighlightSections">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="verticalHeaderHighlightSections">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Add new member</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,1,0">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Search</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditSearchMember">
                   <property name="focusPolicy">
                    <enum>Qt::ClickFocus</enum>
                   </property>
                   <property name="maxLength">
                    <number>32</number>
                   </property>
                   <property name="placeholderText">
                    <string>At least 3 characters</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Found members:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBoxSearchMember"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonAddMember">
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/application.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>onBtnAddMemberClicked()</slot>
  <slot>onLineEditSeachtReturnPressed()</slot>
 </slots>
</ui>
